pghoard/walreceiver.py

```diff
@@ -0,0 +1,206 @@
+from io import BytesIO
```

Comment:
Added header, doesn't show in this comment though since it's added on a line above this.